phpcs.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<ruleset name="Updates API Inspector">
+    <description>Ruleset for Updates API Inspector.</description>
+
+    <!-- Show progress and sniff codes in all reports -->
+    <arg value="ps"/>
+
+    <rule ref="WordPress-Core">
+        <exclude name="WordPress.Files.FileName.InvalidClassFileName" />
```

Comment:
Actually, `InvalidClassFileName` should not be excluded.  When I get around to doing the refactoring to my normal plugin structure, the main `plugin.php` file will be the only one that violates that...so the inline `phpcs:ignore` in that file is just fine.